scipy/stats/stats.py

```diff
@@ -2837,23 +2844,31 @@ def median_absolute_deviation(x, axis=0, center=np.median, scale=1.4826,
     >>> x.std()
     0.9973906394005013
     >>> stats.median_absolute_deviation(x)
-    1.2280762773108278
+    0.82832610097857
```

Comment:
these values needed to change b/c of the change in default `scale` arg, similarly in unit tests.